ChallengesWithTestsMarkVII/ChallengesSet03.cs

```diff
@@ -1,53 +1,81 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace ChallengesWithTestsMarkVII
 {
     public class ChallengesSet03
     {
         public bool ArrayContainsAFalse(bool[] vals)
         {
-            throw new NotImplementedException();
+            if (vals.Contains(false))
```

Comment:
This is good but could be simplified.  This if-statement reads like "if true then return true; if false then return false". Consider just return the result of the Contains method like `return vals.Contains(false);`